package.json

```diff
@@ -23,6 +23,7 @@
     "react-router-dom": "^6.0.2",
     "react-scripts": "^0.9.5",
     "redux": "^4.1.2",
+    "redux-devtools-extension": "^2.13.9",
```

Comment:
- [ ] **(OPTIONAL)** This is totally optional really, but remember that using this extension in your code means that any person who installs it is able to see your stores. In a real life project this could be bad, just remember this in the future.